src/Core/GraphQl/Subscription/SubscriptionManager.php

```diff
@@ -92,7 +92,7 @@ public function getPushPayloads($object): array
         foreach ($subscriptions as [$subscriptionId, $subscriptionFields, $subscriptionResult]) {
             $resolverContext = ['fields' => $subscriptionFields, 'is_collection' => false, 'is_mutation' => false, 'is_subscription' => true];
 
-            $data = ($this->serializeStage)($object, $resourceClass, 'update', $resolverContext);
+            $data = ($this->serializeStage)($object, $resourceClass, 'update_subscription', $resolverContext);
```

Comment:
Why?

Reply:
In the resource metadata I'm creating the Subscription operation, having that name.